dist/dist.py

```diff
@@ -2,21 +2,18 @@
 
 # source_files --
 #    Return a list of the WiredTiger source file names.
-def source_files(skip_includes=False):
-    if not skip_includes:
-        for line in glob.iglob('../src/include/*.[hi]'):
-            yield line
```

Comment:
It looks like we only use this in `style.py` -- it still needs to pick up the header / inline files, I think.